examples/StandardRTLSTag_TWR/StandardRTLSTag_TWR.ino

```diff
@@ -256,7 +260,7 @@ void loop() {
                 }
                 
                 /* Sleep until next blink to save power */
-                DW1000Ng::deepSleep();
```

Comment:
Shouldn't this remain deepSleep?

Reply:
sleep modality depends on the configuration that was applied before

Reply:
Oh ok, i read now deepsleep is removed as a method.